src/AzSK.ADO/Framework/Helpers/AdministratorHelper.ps1

```diff
@@ -3,6 +3,7 @@ class AdministratorHelper{
     static [bool] $isCurrentUserPA=$false;
     static $AllPCAMembers = @()
     static $AllPAMembers = @()
+    static $AllBAMembers = @()
```

Comment:
does it not needed clearing code?

Reply:
already being cleared in GetTotalBAMembers method